lib/elixir_search_extractor/file_upload/csv_uploader.ex

```diff
@@ -0,0 +1,28 @@
+defmodule ElixirSearchExtractor.FileUpload.CsvUploader do
+  import Ecto.Query, warn: false
```

Comment:
Just wondering what is the warning 💭

Reply:
It was actually unnecessary. Removed it from all the files.